its/ruling/src/test/resources/expected/python-S2638.json

```diff
@@ -91,6 +91,9 @@
 'project:numpy-1.16.4/numpy/ma/core.py':[
 6170,
 ],
+'project:numpy-1.16.4/tools/c_coverage/c_coverage_report.py':[
+34,
```

Comment:
Pygments API changed since version 2.12 where the outline parameter was dropped. The previous version where the outline parameter existed is 2.11.2 and was released in January 2022. Not sure is we should do the same as with PyYAML or as this library is less used we could accept this FP.